elements/pfe-tabs/src/pfe-tab-panel.scss

```
@@ -1,11 +1,17 @@
 @import "../../pfe-sass/pfe-sass";
+@import "../../../docs/themes/kube/assets/src/sass/_vars.scss";
+
 
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (focus: pfe-var(link),
   panel: ( // to avoid collisions with tab header
     BackgroundColor: transparent,
     Padding: pfe-var(container-spacer),
+    PaddingTop: $space-48,
```

```suggestion
    PaddingTop: calc(#{pfe-var(container-padding)} * 3),
```

I'd suggest replacing instances using $space-48 with the above example multiplying the container-padding variable by 3, giving us a value of 48px